packages/reassure-cli/src/commands/init.ts

```diff
@@ -0,0 +1,95 @@
+import { mkdirSync, copyFileSync, existsSync, readFileSync, appendFileSync } from 'fs';
+import type { CommandModule } from 'yargs';
+import type { DefaultOptions } from '../types';
+
+import { RESULTS_DIRECTORY, GITIGNORE_DATA } from '../constants';
+import { printBye, printCiSetupHint, printError, printHello, printLog, printWarn } from '../utils/printer';
+
+type InitOptions = {
+  jsStandard?: string;
+} & DefaultOptions;
+/**
+ * @param args arguments which come from the CLI command when ran
+ *
+ * Main runner function for the init command for reassure-cli. It will create the .reassure directory
+ * and copy the test shell script template over there. It will additionally copy over the dangerfile
+ * template to the root folder and update .gitignore file if present and not already containing mentions
+ * of reassure within it.
+ */
+async function run(args: InitOptions): Promise<void> {
+  /**
+   * reassinging printing functions for ease of use since args.logLevel is always required
+   * to properly handle the logLevel option coming from the command.
+   */
+  const log = (...logs: string[]): void => printLog(args.logLevel, ...logs);
+  const warn = (...logs: string[]): void => printWarn(args.logLevel, ...logs);
+  const error = (...logs: string[]): void => printError(args.logLevel, ...logs);
+
+  printHello(args.logLevel);
+
+  log('Checking if reassure setup exists');
+  if (await existsSync(RESULTS_DIRECTORY)) {
+    error('Reassure has already been initialized => exiting');
+    return;
+  }
+
+  try {
+    log('Creating the results directory');
```

Comment:
This step is not needed, as test code will automatically create this directory on first run.